### ISSUE TITLE
Save files paths/Loading old save files on Windows 10

### ISSUE THREAD

**Issue by Zorakknight** [Kitteh6660/Corruption-of-Champions-Mod#341]
I'm having problems loading save files from 2 weeks ago. I got the files here with me and it seems they are not corrupted, as i have both the .sol files as the save standalone. CoC is refusing to load the save standalone and both firefox and chrome do not find the save files, even if they are in their right folders. Both firefox and Chorme did not have their cookies erased, nor was the savefiles or the program itself put on isolation by the anti-virus. Help pls

**Comment by SpearM3064** [comment]
It isn't Windows 10 that's causing the problem, as far as I can tell.  It seems that an update to the Flash Player has changed the save location from ...\localhost to ...\#localWithNet.

This affects CoC, TiTS, and both the CoC and TiTS save editor.

**Comment by gillnavisingh** [comment]
I've only ever saved to file and kept them in the same directory as the downloaded swf
Like this:
(X:/.../CoC/game.swf)
(X:/.../CoC/save_1) <--- no file extension, naturally

Now the game won't load any of my old saves - does that mean I need to convert them to sol?

**Comment by brrritssocold** [comment]
Has this issue been resolved?

**Comment by SpearM3064** [comment]
No, and I'm not sure how it CAN be resolved.  This issue is the result of
an update to Adobe Flash nearly 8 months ago, not a problem with CoC.

The default Save/Load location USED TO BE ...\localhost, but after the
Flash update, it is now ...\#LocalWithNet.

It's probably better to write a tiny tool that migrates all the save files
over to \#LocalWithNet than to try to override the Save/Load behavior.
